pgd_search/tests.py

```diff
@@ -857,6 +858,49 @@ def test_saving_image(self):
         response = self.driver.find_element_by_id("button-save").click()
 
 class ViewTest(TestCase):
-    def home_page_noerror(self):
+
+
+    #Method must start with "test"
+    def test_home_page_noerror(self):
         response = self.client.get(reverse('/'))
         self.assertEqual(response.status_code, 200)
+
+
+
+class CheckDumpTest(TestCase) :
+
+
+    fixtures = ['pgd_search.json']
+
+    def test_download_tsv(self):
+
+        search = Search(segmentLength=5)
+
+        # create associated Search_residues
+        data = {}
+        #data['index'] = 5
+        data['residues'] = 5
+        data['occm_-1'] = 0.58
```

Comment:
Having all six values being 0.58 makes it hard to determine whether the columns are included in the correct order.  Also, how do you know the output is in the correct column?